### PR TITLE
checkup vmlatency: Login to guest console

### PR DESCRIPTION
**Pull request by EdDev**

### CODE REVIEW

**Review by orelmisan — APPROVED** [review]
Thank you!

**Comment by EdDev** [comment]
change: Cleanup a bit the login commit (mainly the `console` package), hack the VMI interface definition to use bridge binding with a specific NAD name and set an automation target to deploy the checkup (register the permissions part).

The PR still needs a basic sanity test to be set (and run), showing this works and allowing the continuation work.

File: .github/workflows/checkup-kubevirt-vm-latency.check.yaml [hunk 1]
```diff
@@ -71,3 +71,29 @@ jobs:
...
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+      - name: Build kiagnose image
+        run: ./automation/make.sh --build-core --build-core-image
+      - name: Build checkup image
+        working-directory: ./checkups/kubevirt-vm-latency
+        run: ./automation/make.sh --build-checkup --build-checkup-image
+      - name: Start cluster
+        run: ./automation/make.sh --e2e -- --install-kind --install-kubectl --create-cluster --deploy-kiagnose
```
Comment (orelmisan) [inline .github/workflows/checkup-kubevirt-vm-latency.check.yaml:89]:
```suggestion
run: ./checkups/kubevirt-vm-latency/automation/make.sh --e2e -- --install-kind --install-kubectl --create-cluster --deploy-kiagnose
```

File: .github/workflows/checkup-kubevirt-vm-latency.check.yaml [hunk 2]
```diff
@@ -71,3 +71,29 @@ jobs:
...
+      - name: Deploy kiagnose
+        run: ./automation/make.sh --e2e -- --deploy-kiagnose
```
Comment (orelmisan) [inline .github/workflows/checkup-kubevirt-vm-latency.check.yaml:91]:
```suggestion
 run: ./checkups/kubevirt-vm-latency/automation/make.sh --e2e -- --deploy-kiagnose
```

File: .github/workflows/checkup-kubevirt-vm-latency.check.yaml [hunk 3]
```diff
@@ -71,3 +71,29 @@ jobs:
...
+      - name: Deploy kubevirt, CNAO and the NetworkAttachementDefinition
+        run: ./checkups/kubevirt-vm-latency/automation/make.sh --e2e -- --deploy-kubevirt --deploy-cnao --define-nad
+      - name: Deploy VM latency checkup
+        run: ./checkups/kubevirt-vm-latency/automation/make.sh --e2e -- --deploy-checkup
+      - name: Run e2e tests
+        run: ../checkups/kubevirt-vm-latency/automation/make.sh --e2e -- --run-tests
```
Comment (orelmisan) [inline .github/workflows/checkup-kubevirt-vm-latency.check.yaml:97]:
```suggestion
run: ./checkups/kubevirt-vm-latency/automation/make.sh --e2e -- --run-tests
```

File: .github/workflows/checkup-kubevirt-vm-latency.check.yaml [hunk 4]
```diff
@@ -71,3 +71,29 @@ jobs:
...
+      - name: Delete cluster
+        run: ./automation/make.sh --e2e -- --delete-cluster
```
Comment (orelmisan) [inline .github/workflows/checkup-kubevirt-vm-latency.check.yaml:99]:
```suggestion
run: ./checkups/kubevirt-vm-latency/automation/make.sh --e2e -- --delete-cluster
```

**Review by orelmisan — APPROVED** [review]
Thank you!

**Pull request merged**